src/textual/widgets/_markdown.py

```diff
@@ -146,10 +147,14 @@ def build_from_token(self, token: Token) -> None:
         self._token = token
         style_stack: list[Style] = [Style()]
         content = Text()
+        repeating_whitespace = re.compile(r"[ \t]+")  # Space or tabs get reduced.
```

Comment:
Would it be worth it (performance-wise) to compile this in the module scope and then use it here?